doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1220 @@
+    * :ref:`record_launch_app`
+    * :ref:`nfc_shell`
+    * :ref:`nrf-nfc-system-off-sample`
+    * :ref:`nfc_tnep_tag`
+    * :ref:`writable_ndef_msg`
+
+nRF5340 samples
+---------------
+
+* Removed the ``nRF5340: Multiprotocol RPMsg`` sample.
+  Use the :ref:`ipc_radio` application instead.
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Added:
+
+    * Loading of radio trims and a fix of a hardware errata for the nRF54H20 SoC to improve the RF performance.
+    * Support for the :ref:`nRF2220 front-end module <ug_radio_fem_nrf2220ek>`.
+    * Workaround for the hardware errata HMPAN-216 for the nRF54H20 SoC.
+
+PMIC samples
+------------
+
+* Added:
+
+  * The :ref:`npm2100_one_button` sample that demonstrates how to support wake-up, shutdown, and user interactions through a single button connected to the nPM2100 PMIC.
+  * The :ref:`npm2100_fuel_gauge` sample that demonstrates how to calculate the battery state of charge of primary cell batteries using the :ref:`nrfxlib:nrf_fuel_gauge`.
+
+* :ref:`npm1300_fuel_gauge` sample:
+
+  * Updated to accommodate API changes in nRF Fuel Gauge library v1.0.0.
+
+SDFW samples
+------------
+
+* Removed the SDFW: Service Framework Client sample as all services demonstrated by the sample have been removed.
+
+SUIT samples
+------------
+
+* Added the :ref:`nrf54h_suit_ab_sample` sample that demonstrates how to perform A/B updates using SUIT manifests.
+
+ * :ref:`nrf54h_suit_sample` sample:
+
+   * Updated:
+
+     * The memory maps to cover the entire available MRAM memory.
+     * The memory maps to place recovery firmware on lower addresses than the main firmware.
+     * By enabling secure entropy source in all main Bluetooth-enabled sample variants (except recovery firmware).
+     * By extending the manifests to process the ``suit-payload-fetch`` sequence of the Nordic top update candidate.
+     * By extending the manifests with build-time checks for consistency between MPI and envelope signing configuration.
+
+Trusted Firmware-M (TF-M) samples
+---------------------------------
+
+* :ref:`tfm_psa_template` sample:
+
+  * Added support for the following attestation token fields:
+
+    * Profile definition
+    * PSA certificate reference (optional), configured using the ``SB_CONFIG_TFM_OTP_PSA_CERTIFICATE_REFERENCE`` sysbuild Kconfig option
+    * Verification service URL (optional), configured using the :kconfig:option:`CONFIG_TFM_ATTEST_VERIFICATION_SERVICE_URL` Kconfig option
+
+* :ref:`tfm_secure_peripheral_partition` sample:
+
+  * Updated documentation with information about how to access other TF-M partitions from the secure partition.
+
+Thread samples
+--------------
+
+* :ref:`ot_cli_sample` sample:
+
+  * Removed support for the nRF54H20 DK.
+
+Zigbee samples
+--------------
+
+* Removed all Zigbee samples.
+  They are now available as separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories.
+
+Wi-Fi samples
+-------------
+
```

Comment:
* Added: 
 
  * Added support for ``nrf7002eb2`` with nRF54 Series devices in the following samples:
 
    * :ref:`wifi_station_sample`
    * :ref:`wifi_scan_sample`
    * :ref:`wifi_shell_sample`
    * :ref:`wifi_radio_test_sd`
    * :ref:`wifi_radio_test`
    * :ref:`wifi_softap_sample`
    * :ref:`wifi_shutdown_sample`
    * :ref:`wifi_twt_sample`
    * :ref:`wifi_offloaded_raw_tx_packet_sample`
    * :ref:`wifi_raw_tx_packet_sample`
    * :ref:`_ble_wifi_provision`

Reply:
Related to https://github.com/nrfconnect/sdk-nrf/pull/21392.

Reply:
Added